_projects/assign_issues.md

```diff
@@ -2,7 +2,8 @@
 collaborating_projects:
   - coala
   - GitMate
-desc: "Make sure that developers get notified about new issues that need their input: Use the tags of the new issue to find developers that have previously solved similar Issues."
+desc: "Make sure that developers get notified about new issues that need their input: Use the tags of the new issue to
```

Comment:
no dude, this needs to follow the 120 rule

Reply:
same for your other pointers

Reply:
No, @sks444 is correct -- see my previous review.

Where the file is being modified, it should be reduced to 80cpl.

There are other lines in this file which are 120cpl which do not need to be modified, so they are not being modified.

Reply:
No worries -- it is a bit odd, what we are doing, but the eventual goal is 80cpl for everything.
But doing that all in one go would be way too disruptive to the repo, and way too much work.
git works better when some parts of the file are not modified, otherwise its diff algorithms cant figure it out, and they even stop using the empty lines as anchor points to keep the diff aligned.